src/index.html

```diff
@@ -22,6 +22,319 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="MOYO"
+          class="logo__image"
+        />
+      </a>
+      <nav class="nav">
```

Comment:
Consider adding space between all elements on one level of depths

Reply:
I accidentally deleted the comment about 'ul' 'li'. Is this really necessary? In previous tasks, mentors confirmed 'div'.